net/golioth/golioth.c

```diff
@@ -90,6 +90,18 @@ static int golioth_setsockopt_dtls(struct golioth_client *client, int sock,
 		}
 	}
 
+	/* If Connection IDs are enabled, set socket option to send CIDs, but not require that the
+	 * server sends one in return.
+	 */
+#ifdef CONFIG_MBEDTLS_SSL_DTLS_CONNECTION_ID
+	int ENABLED = 1;
```

Comment:
I don't like `ENABLED` in uppercase, since I haven't seen such pattern yet. I'd rather use lowercase, just as any other variables and leave uppercase symbols for C preprocessor macros.